docs/UserGuide.md

```diff
@@ -169,93 +175,104 @@ You can find all your activities on a specific date in AtheltiCLI.
 
 **Syntax**
 
 * `set-activity-goal sport/SPORT type/TYPE period/PERIOD target/TARGET`
 
 **Parameters**
 
-* SPORT: The sport for which you want to set a goal. It must be one of the following: run, swim, cycle, general.
-* TYPE: The metric for which you want to set a goal. It must be one of the following: distance, duration.
-* PERIOD: The period for which you want to set a goal. It must be one of the following: daily, weekly, monthly, 
-  yearly. Only activities that are recorded within the period will be counted towards the goal.
-* TARGET: The target value. It must be a positive number. For distance, it is in meters. For duration, it is in
-  minutes.
+* SPORT: The sport for which to set a goal. Options: running, cycling, swimming, general.
+* TYPE: The metric for the goal. Options: distance, duration.
+* PERIOD: The period for the goal. Options: daily, weekly, monthly, yearly. Only activities that are recorded within
+  the period will be counted towards the goal.
+* TARGET: The target value. It must be a positive number. For distance, in meters. For duration, in minutes.
 
 **Examples**
+
 * `set-activity-goal sport/running type/distance period/weekly target/10000` Sets a goal of running 10km per week.
-* `set-activity-goal sport/swimming type/duration period/monthly target/120` Sets a goal of swimming for 2 hours per
-  month.
+* `set-activity-goal sport/swimming type/duration period/monthly target/120` Sets a goal of swimming for 2 hours 
+  per month.
+
+---
 
 ### ✍️ Editing Activity Goals:
 
 `edit-activity-goal`
 
-You can edit your already set goals by mentioning the sport, target, and period of the goal you want to edit.
+You can edit your set goals by specifying the sport, target, and period.
 
 **Syntax**
 
-* `edit-activity-goal sport/SPORT target/TARGET period/PERIOD value/VALUE`
+* `edit-activity-goal sport/SPORT target/TARGET period/PERIOD target/TARGET`
 
 **Parameters**
 
-* SPORT: The sport for which you want to set a goal. It must be one of the following: running, swimming, cycling, general.
-* TARGET: The target for which you want to set a goal. It must be one of the following: distance, duration.
-* PERIOD: The period for which you want to set a goal. It must be one of the following: daily, weekly, monthly, yearly.
-* VALUE: The value of the target. It must be a positive number. For distance, it is in meters. For duration, it is in minutes.
+* TARGET: The new target value. For distance (in meters), for duration (in minutes).
+* See [setting activity goals](#-setting-activity-goals) for the other parameters.
 
 **Examples**
 
 * `edit-activity-goal sport/running type/distance period/weekly target/20000` Edits the goal of running 20km per week.
-* `edit-activity-goal sport/swimming type/duration period/monthly target/60` Edits the goal of swimming for 1 hour per month.
+* `edit-activity-goal sport/swimming type/duration period/monthly target/60` Edits the goal of swimming for 1 hour 
```

Comment:
```suggestion
* `edit-activity-goal sport/swimming type/duration period/monthly target/60` Adjusts the goal of swimming duration to 1 hour 
```
Something like that to clarify that we are actually changing the target value and not filtering by this